<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPaket</class>
 <widget class="QWidget" name="FormPaket">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>950</width><height>700</height></rect>
  </property>
  <property name="windowTitle"><string>Data Paket</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title"><string>Input Paket</string></property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0"><widget class="QLabel" name="label"><property name="text"><string>ID Paket</string></property></widget></item>
      <item row="0" column="1"><widget class="QLineEdit" name="EditId"><property name="placeholderText"><string>auto</string></property><property name="readOnly"><bool>true</bool></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_2"><property name="text"><string>Nama Paket</string></property></widget></item>
      <item row="1" column="1"><widget class="QLineEdit" name="EditNama"/></item>
      <item row="2" column="0"><widget class="QLabel" name="label_3"><property name="text"><string>Harga</string></property></widget></item>
      <item row="2" column="1"><widget class="QDoubleSpinBox" name="SpinHarga"><property name="maximum"><double>9999999999.00</double></property><property name="decimals"><number>2</number></property></widget></item>
      <item row="3" column="0"><widget class="QLabel" name="label_4"><property name="text"><string>Deskripsi</string></property></widget></item>
      <item row="3" column="1"><widget class="QTextEdit" name="EditDeskripsi"/></item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayoutBtns">
        <item><widget class="QPushButton" name="BtnSimpan"><property name="text"><string>Simpan</string></property></widget></item>
        <item><widget class="QPushButton" name="BtnUbah"><property name="text"><string>Ubah</string></property></widget></item>
        <item><widget class="QPushButton" name="BtnHapus"><property name="text"><string>Hapus</string></property></widget></item>
        <item><widget class="QPushButton" name="BtnBersih"><property name="text"><string>Bersih</string></property></widget></item>
        <item><spacer name="horizontalSpacer"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
        <item><widget class="QPushButton" name="btnCetak"><property name="text"><string>Cetak</string></property></widget></item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSearch">
     <item><widget class="QLabel" name="labelCari"><property name="text"><string>Cari</string></property></widget></item>
     <item><widget class="QLineEdit" name="lineCari"><property name="placeholderText"><string>cari nama paket...</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblPaket">
     <property name="columnCount"><number>4</number></property>
     <column><property name="text"><string>ID</string></property></column>
     <column><property name="text"><string>Nama Paket</string></property></column>
     <column><property name="text"><string>Harga</string></property></column>
     <column><property name="text"><string>Deskripsi</string></property></column>
     <property name="selectionBehavior"><enum>QAbstractItemView::SelectRows</enum></property>
     <property name="editTriggers"><set>QAbstractItemView::NoEditTriggers</set></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
